src/pages/projects/introduction-to-cloudflare-workers/index.tsx

```diff
@@ -0,0 +1,342 @@
+      },
+      {
+        title:
+          "Generate New Cloudflare Workers Projects with Wrangler's generate Command",
+        slug:
+          'cloudflare-generate-new-cloudflare-workers-projects-with-wrangler-s-generate-command',
+        path:
+          '/lessons/cloudflare-generate-new-cloudflare-workers-projects-with-wrangler-s-generate-command',
+      },
+      {
+        title: 'Write Your First Cloudflare Workers Serverless Function',
+        slug:
+          'cloudflare-write-your-first-cloudflare-workers-serverless-function',
+        path:
+          '/lessons/cloudflare-write-your-first-cloudflare-workers-serverless-function',
+      },
+      {
+        title: 'Preview and Publish Your Cloudflare Workers Project',
+        slug: 'cloudflare-preview-and-publish-your-cloudflare-workers-project',
+        path:
+          '/lessons/cloudflare-preview-and-publish-your-cloudflare-workers-project',
+      },
+      {
+        title: 'Render HTML Pages with Cloudflare Workers',
+        slug: 'cloudflare-render-html-pages-with-cloudflare-workers',
+        path: '/lessons/cloudflare-render-html-pages-with-cloudflare-workers',
+      },
+      {
+        title: 'Render Cloudflare Region Data for a Request Using request.cf',
+        slug:
+          'cloudflare-render-cloudflare-region-data-for-a-request-using-request-cf',
+        path:
+          '/lessons/cloudflare-render-cloudflare-region-data-for-a-request-using-request-cf',
+      },
+      {
+        title:
+          'Deploy to a Custom Domain with Cloudflare Wrangler Environments',
+        slug:
+          'cloudflare-deploy-to-a-custom-domain-with-cloudflare-wrangler-environments',
+        path:
+          '/lessons/cloudflare-deploy-to-a-custom-domain-with-cloudflare-wrangler-environments',
+      },
+    ],
+  }
+
+  return (
+    <>
+      <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto">
+        <div className="mt-10 mb-16 text-center">
+          <div className="mb-10">
+            <Image priority src={course.image} height="270" width="270" />
+          </div>
+          <p className="text-lg md:text-2xl leading-6 text-gray-500">
+            Portfolio Project Challenge
+          </p>
+          <h1 className="text-2xl md:text-4xl font-bold mt-2">
+            {course.title}
+          </h1>
+        </div>
+        <ProjectBrief
+          className="pb-12"
+          topic={{
+            name: 'cloudflare',
+            label: 'Project Brief',
+          }}
+        >
+          <Markdown className="prose dark:prose-dark pt-2 sm:text-base text-sm leading-normal text-gray-800 dark:text-gray-200 mt-0">
+            {`You are a developer for a national concert promoter.
+
+Music fans from all over the world visit your site to see when and where the next show is going to be. To save them time when they visit the page, we want to show them upcoming events at a venue close to their location.
```

Comment:
you have to have your markdown start at the beginning of the file in your editor for this to behave as expected. This confused me for a long time...

Looks weird but once you get used to it, it's pretty nice